reconcile/utils/terrascript_aws_client.py

```diff
@@ -224,12 +240,23 @@ def __init__(self, integration: str, integration_prefix: str,
                 version="2.2.0"
             )
 
-            b = Backend("s3",
-                        access_key=config['aws_access_key_id'],
-                        secret_key=config['aws_secret_access_key'],
-                        bucket=config['bucket'],
-                        key=config['{}_key'.format(integration)],
-                        region=config['region'])
+            for account in filtered_accounts:
+                terraform_state = account.get("terraform_state")
+                if not terraform_state:
```

Comment:
Would suggest to move declaration of:
- access_key
- secret_key

out of if statement, cause they are the same for both cases.